pages/project/schema/index.js

```diff
@@ -35,12 +52,32 @@ const schema = {
   },
   transferredOutDate: {
     show: true
+  },
+  establishment: {
+    toCSVString: est => est.name,
+    title: 'Primary establishment'
+  },
+  species: {
+    title: 'Animal types',
+    toCSVString: species => species && species.length ? species.join(', ') : '-'
+  },
+  issueDate: {
+    title: 'Grant date',
+    toCSVString: formatCSVDate
+  },
+  raDate: {
+    title: 'RA due date',
+    toCSVString: formatCSVDate
   }
 };
 
-const getSchema = status => {
+const getSchema = (status, csv) => {
   const inactiveLicenceHolder = { licenceHolder: { label: 'Name' } };
 
+  if (csv) {
+    return merge({}, pick(schema, 'licenceNumber', 'title', 'licenceHolder', 'establishment', 'status', 'issueDate', 'amendedDate', 'expiryDate', 'species', 'raDate'));
```

Comment:
What fields is this removing? I'm struggling to think why we would have a field in the online table and _not_ in the CSV?

Reply:
there's a list of fields on the ticket, createdAt and updatedAt are not on it (we use last amended date), also ID, probably others

Reply:
🤔  seems weird to have things in the digital view and NOT in the csv, but whatevs.